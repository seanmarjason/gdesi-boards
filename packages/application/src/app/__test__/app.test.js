import { render } from '@testing-library/react'
import App from '../app';

const session = {
  user: {
    email: 'test@gdesi.io'
  }
}
 
test('App renders correctly', () => {
  const { container } = render(<App session={session} />)
  expect(container).toMatchSnapshot()
})
