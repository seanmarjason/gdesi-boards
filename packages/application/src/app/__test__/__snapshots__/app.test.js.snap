// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionFixed mui-fixed css-5bgt7v-MuiPaper-root-MuiAppBar-root"
    style="--Paper-shadow: 0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12); --Paper-overlay: linear-gradient(rgba(255, 255, 255, 0.092), rgba(255, 255, 255, 0.092));"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-5c1adp-MuiContainer-root"
    >
      <div
        class="MuiToolbar-root MuiToolbar-dense css-10po6k9-MuiToolbar-root"
      >
        <div
          class="MuiBox-root css-1rw111m"
        >
          <a
            class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-19ap7x-MuiTypography-root-MuiLink-root"
            href="/"
          >
            <img
              alt="The house from the offer."
              class="MuiBox-root css-ndi7rl"
              src="desi-sm-to2.png"
            />
          </a>
          <div
            class="MuiBox-root css-1guk29"
          />
        </div>
        <div
          class="MuiBox-root css-h10f7z"
        >
          <a
            class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedSecondary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-colorSecondary MuiButton-root MuiButton-outlined MuiButton-outlinedSecondary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-colorSecondary css-1adhyi6-MuiButtonBase-root-MuiButton-root"
            href="/boards"
            id=":r0:"
            tabindex="0"
          >
            <span
              class="MuiButton-icon MuiButton-startIcon MuiButton-iconSizeMedium css-1v1j2p3-MuiButton-startIcon"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
                data-testid="DashboardIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M3 13h8V3H3zm0 8h8v-6H3zm10 0h8V11h-8zm0-18v6h8V3z"
                />
              </svg>
            </span>
            Boards
          </a>
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary css-7t8udx-MuiButtonBase-root-MuiButton-root"
            id=":r1:"
            tabindex="0"
            type="button"
          >
            Sign out
          </button>
        </div>
        <div
          class="MuiBox-root css-zkjl6t"
        >
          <button
            aria-label="Menu button"
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-1a769d0-MuiButtonBase-root-MuiIconButton-root"
            id=":r2:"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
              data-testid="MenuIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M3 18h18v-2H3zm0-5h18v-2H3zm0-7v2h18V6z"
              />
            </svg>
          </button>
        </div>
      </div>
    </div>
  </header>
  <div
    class="MuiBox-root css-w0zn1l"
    id="hero"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1f9chyi-MuiContainer-root"
    >
      <div
        class="MuiStack-root css-i5068v-MuiStack-root"
      >
        <h1
          class="MuiTypography-root MuiTypography-h1 css-163wgjs-MuiTypography-root"
        >
          Remote & Hybrid Working Done Right!
        </h1>
        <p
          class="MuiTypography-root MuiTypography-body1 css-11qfyl7-MuiTypography-root"
        >
          Explore our latest tools to support your remote or hybrid team to collaborate across borders and timezones.
        </p>
        <div
          class="MuiBox-root css-1fmaocf"
        >
          <a
            class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary css-7t8udx-MuiButtonBase-root-MuiButton-root"
            href="/sign-up"
            id=":r3:"
            tabindex="0"
          >
            Register
          </a>
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorSecondary MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorSecondary css-1yxs6e2-MuiButtonBase-root-MuiButton-root"
            id=":r4:"
            tabindex="0"
            type="button"
          >
            Sign in
          </button>
        </div>
      </div>
    </div>
  </div>
  <div>
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-xkgqcd-MuiContainer-root"
      id="features"
    >
      <div
        class="MuiBox-root css-1cxhys7"
      >
        <h2
          class="MuiTypography-root MuiTypography-h4 MuiTypography-gutterBottom css-1ql9l6j-MuiTypography-root"
        >
          Product features
        </h2>
        <p
          class="MuiTypography-root MuiTypography-body1 css-1boiojk-MuiTypography-root"
        >
          Experience the latest features to improve how you work remotely.
        </p>
      </div>
      <div
        class="MuiBox-root css-1f9uce6"
      >
        <div>
          <div
            class="MuiBox-root css-ltn6y3"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorPrimary MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorPrimary MuiBox-root css-yu57db-MuiButtonBase-root-MuiButton-root"
              id=":r5:"
              tabindex="0"
              type="button"
            >
              <div
                class="MuiBox-root css-13trncf"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
                  data-testid="DashboardIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M3 13h8V3H3zm0 8h8v-6H3zm10 0h8V11h-8zm0-18v6h8V3z"
                  />
                </svg>
                <h6
                  class="MuiTypography-root MuiTypography-h6 css-iphlls-MuiTypography-root"
                >
                  Team Task Boards
                </h6>
                <p
                  class="MuiTypography-root MuiTypography-body2 css-10v0jpn-MuiTypography-root"
                >
                  Collaborate with your team asynchronously, continuously capturing the latest updates on your work in a central location.
                </p>
              </div>
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorPrimary MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorPrimary MuiBox-root css-1pd9oqd-MuiButtonBase-root-MuiButton-root"
              id=":r6:"
              tabindex="0"
              type="button"
            >
              <div
                class="MuiBox-root css-f1j5jw"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
                  data-testid="HowToRegIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="m9 17 3-2.94c-.39-.04-.68-.06-1-.06-2.67 0-8 1.34-8 4v2h9zm2-5c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4m4.47 8.5L12 17l1.4-1.41 2.07 2.08 5.13-5.17 1.4 1.41z"
                    fill-rule="evenodd"
                  />
                </svg>
                <h6
                  class="MuiTypography-root MuiTypography-h6 css-iphlls-MuiTypography-root"
                >
                  Check Ins
                </h6>
                <p
                  class="MuiTypography-root MuiTypography-body2 css-10v0jpn-MuiTypography-root"
                >
                  Preiodically check in with your team manager to showcase your progress, and highlight your achievements or where you might need support.
                </p>
              </div>
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorPrimary MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorPrimary MuiBox-root css-1pd9oqd-MuiButtonBase-root-MuiButton-root"
              id=":r7:"
              tabindex="0"
              type="button"
            >
              <div
                class="MuiBox-root css-f1j5jw"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
                  data-testid="SmartToyIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M20 9V7c0-1.1-.9-2-2-2h-3c0-1.66-1.34-3-3-3S9 3.34 9 5H6c-1.1 0-2 .9-2 2v2c-1.66 0-3 1.34-3 3s1.34 3 3 3v4c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2v-4c1.66 0 3-1.34 3-3s-1.34-3-3-3M7.5 11.5c0-.83.67-1.5 1.5-1.5s1.5.67 1.5 1.5S9.83 13 9 13s-1.5-.67-1.5-1.5M16 17H8v-2h8zm-1-4c-.83 0-1.5-.67-1.5-1.5S14.17 10 15 10s1.5.67 1.5 1.5S15.83 13 15 13"
                  />
                </svg>
                <h6
                  class="MuiTypography-root MuiTypography-h6 css-iphlls-MuiTypography-root"
                >
                  Two-way communication with OpenAI
                </h6>
                <p
                  class="MuiTypography-root MuiTypography-body2 css-10v0jpn-MuiTypography-root"
                >
                  Harness the power of OpenAI to simulate two-way comnunication, ensuring you give the right amount of information to your team members when they need it.
                </p>
              </div>
            </button>
          </div>
        </div>
        <div
          class="MuiBox-root css-eld7w0"
        >
          <div
            class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-17n8v7v-MuiPaper-root-MuiCard-root"
          >
            <div
              class="MuiBox-root css-j4xxni"
              style="--items-imageLight: url("http://localhost:3000/Dashboard.png"); --items-imageDark: url("http://localhost:3000/Dashboard.png");"
            />
          </div>
        </div>
      </div>
    </div>
    <hr
      class="MuiDivider-root MuiDivider-fullWidth css-x148m1-MuiDivider-root"
    />
    <div
      class="MuiBox-root css-1kbo4vh"
      id="highlights"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-8ghp95-MuiContainer-root"
      >
        <div
          class="MuiBox-root css-fnsx6j"
        >
          <h2
            class="MuiTypography-root MuiTypography-h4 MuiTypography-gutterBottom css-iltb2v-MuiTypography-root"
          >
            Highlights
          </h2>
          <p
            class="MuiTypography-root MuiTypography-body1 css-1n2xv6c-MuiTypography-root"
          >
            Explore why our product stands out: adaptability, scalability, user-friendly design, and innovation.
          </p>
        </div>
        <div
          class="MuiGrid2-root MuiGrid2-container MuiGrid2-direction-xs-row MuiGrid2-spacing-xs-2 css-ubb9a5-MuiGrid2-root"
        >
          <div
            class="MuiGrid2-root MuiGrid2-direction-xs-row MuiGrid2-grid-xs-12 MuiGrid2-grid-sm-6 MuiGrid2-grid-md-4 css-1sbb1tz-MuiGrid2-root"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiCard-root MuiStack-root css-hxm13v-MuiPaper-root-MuiCard-root-MuiStack-root"
              style="--Paper-shadow: none; --Paper-overlay: linear-gradient(rgba(255, 255, 255, 0), rgba(255, 255, 255, 0));"
            >
              <div
                class="MuiBox-root css-yv7cd7"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
                  data-testid="SettingsSuggestRoundedIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="m18.04 7.99-.63-1.4-1.4-.63c-.39-.18-.39-.73 0-.91l1.4-.63.63-1.4c.18-.39.73-.39.91 0l.63 1.4 1.4.63c.39.18.39.73 0 .91l-1.4.63-.63 1.4c-.17.39-.73.39-.91 0m3.24 4.73-.32-.72c-.18-.39-.73-.39-.91 0l-.32.72-.73.32c-.39.18-.39.73 0 .91l.72.32.32.73c.18.39.73.39.91 0l.32-.72.73-.32c.39-.18.39-.73 0-.91zm-5.04 1.65 1.23.93c.4.3.51.86.26 1.3l-1.62 2.8c-.25.44-.79.62-1.25.42l-1.43-.6c-.2.13-.42.26-.64.37l-.19 1.54c-.06.5-.49.88-.99.88H8.38c-.5 0-.93-.38-.99-.88l-.19-1.54c-.22-.11-.43-.23-.64-.37l-1.43.6c-.46.2-1 .02-1.25-.42l-1.62-2.8c-.25-.44-.14-.99.26-1.3l1.23-.93V14c0-.12 0-.25.01-.37l-1.23-.93c-.4-.3-.51-.86-.26-1.3l1.62-2.8c.25-.44.79-.62 1.25-.42l1.43.6c.2-.13.42-.26.64-.37l.19-1.54c.05-.49.48-.87.98-.87h3.23c.5 0 .93.38.99.88l.19 1.54c.22.11.43.23.64.37l1.43-.6c.46-.2 1-.02 1.25.42l1.62 2.8c.25.44.14.99-.26 1.3l-1.23.93c.01.12.01.24.01.37s0 .24-.01.36M13 14c0-1.66-1.34-3-3-3s-3 1.34-3 3 1.34 3 3 3 3-1.34 3-3"
                  />
                </svg>
              </div>
              <div>
                <p
                  class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-1o1m5nr-MuiTypography-root"
                >
                  Self-service
                </p>
                <p
                  class="MuiTypography-root MuiTypography-body2 css-1u2cfu2-MuiTypography-root"
                >
                  Regiter, create your own boards, assign your team members, and start benefiting from asynchronous collaboration.
                </p>
              </div>
            </div>
          </div>
          <div
            class="MuiGrid2-root MuiGrid2-direction-xs-row MuiGrid2-grid-xs-12 MuiGrid2-grid-sm-6 MuiGrid2-grid-md-4 css-1sbb1tz-MuiGrid2-root"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiCard-root MuiStack-root css-hxm13v-MuiPaper-root-MuiCard-root-MuiStack-root"
              style="--Paper-shadow: none; --Paper-overlay: linear-gradient(rgba(255, 255, 255, 0), rgba(255, 255, 255, 0));"
            >
              <div
                class="MuiBox-root css-yv7cd7"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
                  data-testid="ConstructionRoundedIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="m20.99 17.99-4.94-4.94-2.12 2.12 4.94 4.94c.59.59 1.54.59 2.12 0s.58-1.54 0-2.12M17.65 10c1.93 0 3.5-1.57 3.5-3.5 0-.58-.16-1.12-.41-1.6l-2.7 2.7-1.49-1.49 2.7-2.7c-.48-.25-1.02-.41-1.6-.41-1.93 0-3.5 1.57-3.5 3.5 0 .41.08.8.21 1.16l-1.85 1.85-1.78-1.78c.39-.39.39-1.02 0-1.41l-.71-.71 2.12-2.12c-1.17-1.17-3.07-1.17-4.24 0L5.08 6.32c-.39.39-.39 1.02 0 1.41l.71.71H3.25c-.19 0-.37.07-.5.21-.28.28-.28.72 0 1l2.54 2.54c.28.28.72.28 1 0 .13-.13.21-.31.21-.5V9.15l.7.7c.39.39 1.02.39 1.41 0l1.78 1.78-6.35 6.35c-.59.59-.59 1.54 0 2.12.59.59 1.54.59 2.12 0L16.48 9.79c.37.13.76.21 1.17.21"
                  />
                </svg>
              </div>
              <div>
                <p
                  class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-1o1m5nr-MuiTypography-root"
                >
                  Scalable
                </p>
                <p
                  class="MuiTypography-root MuiTypography-body2 css-1u2cfu2-MuiTypography-root"
                >
                  Add more tasks as your workload increases, manage them all in a central place in collaboration with your team.
                </p>
              </div>
            </div>
          </div>
          <div
            class="MuiGrid2-root MuiGrid2-direction-xs-row MuiGrid2-grid-xs-12 MuiGrid2-grid-sm-6 MuiGrid2-grid-md-4 css-1sbb1tz-MuiGrid2-root"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiCard-root MuiStack-root css-hxm13v-MuiPaper-root-MuiCard-root-MuiStack-root"
              style="--Paper-shadow: none; --Paper-overlay: linear-gradient(rgba(255, 255, 255, 0), rgba(255, 255, 255, 0));"
            >
              <div
                class="MuiBox-root css-yv7cd7"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
                  data-testid="ThumbUpAltRoundedIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M13.12 2.06 7.58 7.6c-.37.37-.58.88-.58 1.41V19c0 1.1.9 2 2 2h9c.8 0 1.52-.48 1.84-1.21l3.26-7.61C23.94 10.2 22.49 8 20.34 8h-5.65l.95-4.58c.1-.5-.05-1.01-.41-1.37-.59-.58-1.53-.58-2.11.01M3 21c1.1 0 2-.9 2-2v-8c0-1.1-.9-2-2-2s-2 .9-2 2v8c0 1.1.9 2 2 2"
                  />
                </svg>
              </div>
              <div>
                <p
                  class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-1o1m5nr-MuiTypography-root"
                >
                  Harness the power of AI
                </p>
                <p
                  class="MuiTypography-root MuiTypography-body2 css-1u2cfu2-MuiTypography-root"
                >
                  Improve your communication by harnessing AI to simulate converations with your team, prompting you to provide the right information..
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <hr
      class="MuiDivider-root MuiDivider-fullWidth css-x148m1-MuiDivider-root"
    />
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-17h02ml-MuiContainer-root"
    >
      <div
        class="MuiBox-root css-fdshit"
      >
        <div
          class="MuiBox-root css-5dev9e"
        >
          <div
            class="MuiBox-root css-unvpgn"
          >
            <img
              alt="The house from the offer."
              class="MuiBox-root css-ndi7rl"
              src="desi-sm-to2.png"
            />
            <div
              class="MuiStack-root css-1oxswqj-MuiStack-root"
            />
          </div>
        </div>
        <div
          class="MuiBox-root css-yvp7c"
        />
        <div
          class="MuiBox-root css-yvp7c"
        />
        <div
          class="MuiBox-root css-yvp7c"
        />
      </div>
      <div
        class="MuiBox-root css-1eqvk7p"
      >
        <div />
        <div
          class="MuiStack-root css-11ppy9q-MuiStack-root"
        />
      </div>
    </div>
  </div>
</div>
`;
